CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # Release Notes
 
+## Version 1.19.0-beta.0 - 2021-04-05
+
+Features:
+
+* add minikube image rm command [#10924](https://github.com/kubernetes/minikube/pull/10924)
+* create registry credentials when gcp-auth addon is enabled [#10853](https://github.com/kubernetes/minikube/pull/10853)
+* new command: `minikube cp` to copy files into minikube [#10198](https://github.com/kubernetes/minikube/pull/10198)
+* Add flag "--listen-address" for docker and podman driver [#10653](https://github.com/kubernetes/minikube/pull/10653)
```

Comment:
consistent woring,

since previous one is "new" we can say
new flag ...

Reply:
done